lib/manageiq/graphql/field_name_camelizer.rb

```diff
@@ -0,0 +1,20 @@
+module ManageIQ
+  module GraphQL
+    class FieldNameCamelizer
+      def instrument(_type, field)
+        field.property = field.name.underscore.to_sym if field.resolve_proc.kind_of?(::GraphQL::Field::Resolve::NameResolve)
+        field.name = field.name.camelize(:lower)
+
+        field.arguments = Hash[
```

Comment:
For some reason I had difficulty parsing this at first glance (maybe the combination of the non-destructive map and doing destructive things in the block?). WDYT about:

```ruby
field.arguments.each_value do |argument|
  argument.as = argument.name.underscore
  argument.name = argument.name.camelize(:lower)
end
field.arguments.transform_keys! { |name| field.arguments[name].name }
```